plugins/versionpress/admin/deactivate.php

```diff
@@ -30,26 +32,30 @@ function _vp_button($label, $action, $type = "delete", $cssClass = "") {
     <h2 class="vp-deactivation-header">VersionPress deactivation</h2>
 
 
-
     <div class="error below-h2">
         <p>You are about to deactivate VersionPress. Be aware that:</p>
 
         <ul>
 
             <li>
                 <span class="icon icon-notification warning-color"></span>
-                <span class="vp-highlight-text">Later installations of VersionPress <strong>will not be able to undo or rollback changes</strong> done by this installation.</span> The changes are still technically in the repository if you want to inspect them using e.g. the command line tools etc.
+                <span class="vp-highlight-text">Later installations of VersionPress <strong>will not be able to undo or
```

Comment:
Hard wraps in HTML usually make the editing less pleasant, maybe we could exclude these "PHP files which are essentially HTML files with a little bit of code" from code sniffing. We have only a few of them.